charts/kyverno/values.yaml

```diff
@@ -131,7 +131,7 @@ podAntiAffinity:
           - key: app.kubernetes.io/name
             operator: In
             values:
-            - kyverno
+            - '{{ template "kyverno.fullname" . }}'
```

Comment:
We use `app.kubernetes.io/name: {{ template "kyverno.name" . }}` (not `kyverno.fullname`)